### PR TITLE
Bugfix - Driver Code clean up

### PR DESCRIPTION
**Pull request by ksookdhew**
## Description
Clean up Code for Driver feature 

## Checklist 
- [x] The code follows the project's coding standards. 
- [x] 100% Unit test coverage for View Models 
- [x] All warnings Resolved.

### CODE REVIEW

File: F1/F1/Utilities/ViewController+Extension.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//
+//  ViewController+Extension.swift
+//  F1
+//
+//  Created by Kaitlyn Sookdhew on 2024/05/09.
+//
+
+import Foundation
```
Comment (kieran-woodrow) [inline F1/F1/Utilities/ViewController+Extension.swift:8]: can remove foundations. UIKIt imports it

File: F1/F1/Utilities/ViewController+Extension.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import UIKit
+
+extension UIViewController{
```
Comment (kieran-woodrow) [inline F1/F1/Utilities/ViewController+Extension.swift:11]: nice job here with the extension!

File: F1/F1/Utilities/ViewController+Extension.swift (hunk 1 above)
Comment (kieran-woodrow) [reply F1/F1/Utilities/ViewController+Extension.swift:8]: keep a look out for this everywhere else

**Review by kieran-woodrow — CHANGES_REQUESTED** [review]

File: F1/F1/Utilities/ViewController+Extension.swift (hunk 2 above)
Comment (simekanidvt) [inline F1/F1/Utilities/ViewController+Extension.swift:11]:
```suggestion
extension UIViewController {
```

Comment (kieran-woodrow) [reply F1/F1/Utilities/ViewController+Extension.swift:11]: swiftlint should have picked this us. Please do a pr for going through all the linting errors

**Review by kieran-woodrow — APPROVED** [review]

**Review by giacatano — APPROVED** [review]

**Pull request merged**